dev-lua/luarocks/luarocks-3.2.1.ebuild

```diff
@@ -0,0 +1,34 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="A deployment and management system for Lua modules"
+HOMEPAGE="http://www.luarocks.org"
+SRC_URI="http://luarocks.org/releases/${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~x86"
+IUSE=""
+
+DEPEND="dev-lang/lua:=
+		net-misc/curl
+		dev-libs/openssl:="
```

Comment:
Nice! I ran into a requirement for luarocks >= 3 yesterday, and didn't think to look through the PRs for any new ebuild implementations.

Are these dependencies necessary? I wrote an [ebuild for luarocks 3.3.1](https://git.sr.ht/~laumann/portage/tree/master/dev-lua/luarocks/luarocks-3.3.1.ebuild) without these deps which seems to work.

Reply:
Yes, it uses curl and openssl. Odds are you already have those installed.

Reply:
Ok, thanks, yes they're most likely installed.